### ISSUE TITLE
End-to-end Unsupervised Learning of Long-Term 3D Stable objects. (arXiv:2301.03426v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#24285]
End-to-end Unsupervised Learning of Long-Term 3D Stable objects. (arXiv:2301.03426v1 [cs.CV])<br>
https://ift.tt/BhxcI2O<br>
<p>3D point cloud semantic classification is an important task in robotics as it enables a better understanding of the mapped environment. This work proposes to learn the long-term stability of the 3D objects using a neural network based on PointNet++, where the long-term stable object refers to a static object that cannot move on its own (e.g. tree, pole, building). The training data is generated in an unsupervised manner by assigning a continuous label to individual points by exploiting multiple time slices of the same environment. Instead of using discrete labels, i.e. static/dynamic, we propose to use a continuous label value indicating point temporal stability to train a regression PointNet++ network. We evaluated our approach on point cloud data of two parking lots from the NCLT dataset. The experiments' performance reveals that static vs dynamic object classification is best performed by training a regression model, followed by thresholding, compared to directly training a classification model.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
